src/runtime_src/core/common/shim/hwctx_handle.h

```diff
@@ -55,6 +55,13 @@ class hwctx_handle
   virtual slot_id
   get_slotidx() const = 0;
 
+  // Get number of UCs in the hardware context created
```

Comment:
do we want to this implemented in shim?
xrt (elfloader) doesn't have this info?

Reply:
@xuhz , hardware context should have this information. It knows how many columns in a partition (hardware context). Based on architecture, it knows number of ucs as well.